content/kubermatic/master/architecture/support_policy/provider_support_matrix/kubevirt/kubevirt.en.md

```diff
@@ -21,6 +21,32 @@ KKP supports KubeVirt Operator >= 0.19.0 and the Containerized Data Importer >=
 to run KubeVirt, however a Kubernetes cluster consists of 3 nodes with 2 CPUs, 4GB of RAM and 30GB of storage, to have a
 minimal installation.
 
+### KubeVirt Configuration Requirements
+Kubevirt requires the following configuration to be used with KKP.
```

Comment:
Not a lot but please take care for how it's written, so change here and below to `KubeVirt`.

Reply:
Fixed everywhere. Thanks